alpine-base-php/php7/etc/services/run_fpm

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/with-contenv sh
 set -e
 
-exec /usr/sbin/php-fpm7 -c /etc/php7/php.ini -y /etc/php7/php-fpm.conf -F
+if [ "${PHP_XDEBUG}" = "true" ]; then
+  exec /usr/sbin/php-fpm7 -c /etc/php7/php.ini -d zend_extension=xdebug.so -y /etc/php7/php-fpm.conf -F
```

Comment:
This bit isn't working for me - with PHP_XDEBUG=true in .env, `php -v` shows:
```
PHP 7.3.27 (cli) (built: Feb  4 2021 21:04:33) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.27, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.27, Copyright (c) 1999-2018, by Zend Technologies
```
Adding `RUN echo "zend_extension=xdebug.so" >> /etc/php7/conf.d/30_xdebug.ini` to the end of Dockerfile gives:
```
PHP 7.3.27 (cli) (built: Feb  4 2021 21:04:33) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.27, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.27, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.9.8, Copyright (c) 2002-2020, by Derick Rethans
```

Reply:
The result you got is what I expect to happen; the service run command *only* loads xdebug in the FPM sapi, and you tested the CLI. Dropping in  `/etc/php7/conf.d/30_xdebug.ini` will force the extension to load regardless of env settings, so that would be not helpful for server deploys :-)

If I test the FPM response via an index.php that just contains `phpinfo()` I get:

![Screen Shot 2021-03-03 at 10 01 45 am](https://user-images.githubusercontent.com/148719/109726896-8d2bb800-7c07-11eb-8328-a9f96b039a7a.png)
![Screen Shot 2021-03-03 at 10 02 04 am](https://user-images.githubusercontent.com/148719/109726906-8ef57b80-7c07-11eb-9c95-b8f95f8d2ecc.png)

Reply:
D'oh. I knew I was missing something, but that's more obvious than I'd expected.